iaso/api/serializers.py

```diff
@@ -104,10 +107,14 @@ class Meta:
             "search_index",
             "created_at",
             "org_unit_type_id",
+            "creator",
         ]
 
 
 class OrgUnitSmallSearchSerializer(OrgUnitSerializer):
+    def get_creator(self, org_unit):
+        return None if org_unit.creator is None else org_unit.creator.username
+
```

Comment:
Is it necessary to redefine here if the same code already appears in the parent class?